cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -21,8 +22,8 @@
     {%- endfor %}
 
     {% for block in page.content -%}
-        {% if block.block_type == 'feedback' %}
-            {% include 'setup-feedback-and-render.html' %}
+        {% if block.block_type in ['feedback', 'conference_registration_form'] %}
```

Comment:
Is there any other way to express this check that doesn't have to be updated every time we add a new type of form block?